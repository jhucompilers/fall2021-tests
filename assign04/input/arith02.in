PROGRAM arith02;
  VAR a, b, c : INTEGER;

BEGIN
  a := 7;
  b := 29;

  c := (b - 5 - 3);
  WRITE c;

  c := (b DIV 2);
  WRITE c;

  c := a * 1;
  WRITE c;

  c := (b - 5 - 3) * (b DIV 2);
  WRITE c;

  c := (b - 5 - 3) * (b DIV 2) + a * 1;
  WRITE c;
END.
