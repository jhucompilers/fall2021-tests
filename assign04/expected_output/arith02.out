21
14
7
294
301
